README.md

```diff
@@ -63,7 +63,7 @@ The bootstrap process can be executed either from the bootable system or from a
 
 1. Clone this repository
 2. Set your target IP address in `./conf/bootstrap.coffee`
-3. Edit any other configuration of interest
+3. Edit any other configuration of interest (If needed to re-edit the configuration, do the modification in `./conf/user.yaml`)
```

Comment:
No, the user shall never modify bootstrap, the first time you run nikita-arch, the user.yaml is generated with the value asked at the command line